src/AWS.Logger.Core/Core/AWSLoggerCore.cs

```diff
@@ -662,23 +662,75 @@ private void LogLibraryServiceError(Exception ex, string serviceUrl = null)
         /// <summary>
         /// Write Exception details to the file specified with the filename
         /// </summary>
-        public static void LogLibraryError(Exception ex, string LibraryLogFileName)
+        public static void LogLibraryError(Exception? ex, string LibraryLogFileName)
         {
             try
             {
-                using (StreamWriter w = File.AppendText(LibraryLogFileName))
-                {
-                    w.WriteLine("Log Entry : ");
-                    w.WriteLine("{0}", DateTime.Now.ToString());
-                    w.WriteLine("  :");
-                    w.WriteLine("  :{0}", ex.ToString());
-                    w.WriteLine("-------------------------------");
-                }
+                StringBuilder sb = new StringBuilder(512);
```

Comment:
Looks like something in your tooling has added extra lines through out the change. Can you remove the extra lines?